webui/react/src/App.tsx

```diff
@@ -41,6 +41,7 @@ import css from './App.module.scss';
 
 import 'antd/dist/reset.css';
 import '@hpe.com/glide-data-grid/dist/index.css';
+import 'core-js/actual/structured-clone';
```

Comment:
is there a reason why its declared in App.tsx? if no, i think index.tsx is better cuz its the top level